internal/repositories/devicedefinition/devicedefinition.go

```diff
@@ -0,0 +1,224 @@
+package devicedefinition
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	gmodel "github.com/DIMO-Network/identity-api/graph/model"
+	"github.com/DIMO-Network/identity-api/internal/helpers"
+	"github.com/DIMO-Network/identity-api/internal/repositories/base"
+	"github.com/vektah/gqlparser/v2/gqlerror"
+	"golang.org/x/exp/slices"
+	"net/http"
+	"net/url"
+	"path"
+	"strings"
```

Comment:
std lib vars should be separate from external imports if you are using vscode this can be done by changing the golang formater to` goimports`

Reply:
The references was updated